resources/Microsoft.Windows.Developer/Microsoft.Windows.Developer.psm1

```diff
@@ -969,4 +1124,24 @@ function Restore-GroupPolicyPowerPlanSetting([HashTable[]]$GPRegArray) {
 function Disable-GroupPolicyPowerPlanSetting {
     Remove-Item $GroupPolicyPowerPlanRegistryKeyPath -Recurse -Force | Out-Null
 }
+
+# Convert Firewall Rule name to Escape Wildcard Characters. It will append '[', ']' and '*' with a backtick.
+function ConvertTo-FirewallRuleNameEscapedString {
+    [CmdletBinding()]
+    [OutputType([System.String])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        $Name
+    )
+
+    return $Name.Replace('[', '`[').Replace(']', '`]').Replace('*', '`*')
+}
+
+# Workaround mock issue for Get-NetFirewallPortFilter
```

Comment:
This is detailed in a [pester bug](https://github.com/pester/Pester/issues/2643), for pragmatism working around it.